js/layout-javascript/news-feed-code.js

```diff
@@ -1636,7 +1636,9 @@ DynamicList.prototype.searchData = function(options) {
       }
 
       if (limitEntriesEnabled) {
-        _this.$container.find('.limit-entries-text')[truncated && _this.data.limitEntries > 0 ? 'removeClass' : 'addClass']('hidden');
+        // Do not show limit text when user is searching or filtering
+        var showLimitText = _this.isSearching || _this.isFiltering || _this.showBookmarks;
+        _this.$container.find('.limit-entries-text').toggleClass('hidden', showLimitText);
```

Comment:
@upplabs-alex-levchenko @YaroslavOvdii A new linebreak is needed after a block of variable declaration.

```suggestion

        _this.$container.find('.limit-entries-text').toggleClass('hidden', showLimitText);
```

Reply:
fixed